example/batched_solve/static_pivoting.cpp

```diff
@@ -162,16 +167,16 @@ int main(int /*argc*/, char ** /*argv[]*/) {
     Kokkos::deep_copy(A2, A);
     Kokkos::deep_copy(Y2, Y);
 
-    write3DArrayToMM("A.mm", A);
-    write2DArrayToMM("Y.mm", Y);
+    KokkosKernels::Impl::kk_write_3Dview_to_file(A, "A.txt");
```

Comment:
Do we always write to a file in the directory wherever the executable is run from? This is not common practice for any of our other tests. Do we need to write it or just pushing it to std out is enough?

Reply:
It is only for an example, not for a (unit) test, I can remove that if you prefer.

Reply:
But yes, it writes to a file in the directory where you run the example executable, I did so mostly for users to check the computation if they want.

Reply:
It is ok. We typically don't do it, but that doesn't mean it is wrong. In Trilinos land, we avoid it because it is parallel file write. It shouldn't be a problem in KK.